app/locale/pl.coffee

```diff
@@ -11,13 +11,13 @@ module.exports = nativeDescription: "polski", englishDescription: "Polish", tran
 #    built_for_teachers_subtitle1: "Computer Science"
 #    built_for_teachers_subblurb1: "Starting with our free Introduction to Computer Science course, students master core coding concepts such as while/for loops, functions, and algorithms."
     built_for_teachers_subtitle2: "Tworzenie gier"
-#    built_for_teachers_subblurb2: "Learners construct mazes and use basic input handling to code their own games that can be shared with friends and family."
+#    built_for_teachers_subblurb2: "Studenci tworzą labirynty i używają klawiatury, aby zaprogramować własne gry i dzielić się nimi ze znajomymi i rodziną."
     built_for_teachers_subtitle3: "Tworzenie stron internetowych"
     built_for_teachers_subblurb3: "Używając HTML, CSS i jQuery, uczniowie wykazują się kreatywnością abt stworzyć ich własne strony internetowe z ustalonym adresem URL, który mogą udostępnić go swoim kolegom/koleżankom z klasy."
-#    century_skills_title: "21st Century Skills"
+#    century_skills_title: "Kompetencje 21-ego wieku"
```

Comment:
How about:
```suggestion
#    century_skills_title: "Umiejętności 21-ego wieku"
```

Reply:
If would be nice if you @fatherJS could apply the suggestion above from  @harnash
I also think the `Umiejętności` fits the context better.